core/embed/trezorhal/optiga/optiga_commands.c

```diff
@@ -33,9 +33,17 @@
 #include "sha2.h"
 
 // Static buffer for commands and responses.
-static uint8_t tx_buffer[1750] = {0};
+static uint8_t tx_buffer[OPTIGA_MAX_COMMAND_SIZE] = {0};
```

Comment:
Not directly related to this pull request: I don't like using global static variables. I would suggest defining a structure that contains the buffer and its size, and then passing it explicitly as a parameter.

Reply:
The same applies to the frame status and channel status variables in `optiga_transport.c`.

Reply:
I don't see any use for that, especially given that there is a single Optiga. Even if there were multiple Optigas, it probably wouldn't be a good idea either.

Reply:
My opinion is that what I suggest is a better-structured and modular code that is easier to understand, review and maintain.

I think it is worth doing even if there is a single Optiga.